draft-ietf-quic-ack-frequency.md

```diff
@@ -121,18 +121,17 @@ This document uses terms, definitions, and notational conventions described in
 # Motivation
 
 A receiver acknowledges received packets, but it can delay sending these
-acknowledgements. The delaying of acknowledgements can impact connection
-throughput, loss detection and congestion controller performance at a data
-sender, and CPU utilization at both a data sender and a data receiver.
+acknowledgements. Delaying acknowledgements can impact connection
+throughput, loss detection and congestion controller performance at the
+sender, and CPU utilization at both the sender and receiver.
```

Comment:
Why did you remove "data"? Think this saying data sender is more clear.

Reply:
A sender sends ACK eliciting frames besides data, but I can revert this part of the PR if you want.